livekit-android-sdk/src/main/java/io/livekit/android/room/Room.kt

```diff
@@ -973,6 +980,9 @@ constructor(
         participant: RemoteParticipant,
     ) {
         listener?.onTrackUnsubscribed(track, publication, participant, this)
+        if (e2eeManager != null) {
```

Comment:
```suggestion
        e2eeManager?.let { e2eeManager ->
```

Removes the need for the `!!` in the following line.